labs/crazyegg-integration/metadata.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Crazy Egg Integration
+summary: Crazy Egg integration allows you to see heatmaps with Optimizely data. For more information on how to use this integration, please take a look at this Knowledge Base article: https://help.optimizely.com/Integrate_Optimizely_with_Crazyegg
+revisionDate: '2021-04-05'
+labels:
+  - web
+  - analytics
+  - crazyegg
+  - forms
+  - heatmap
+
+excludeFromListing: false
+
+author:
+  name: Crazy Egg Team
+  headshot: https://w3-assets.crazyeggcdn.com/images/svg/logo-ce-main.svg
```

Comment:
Note: I believe this renders as a circle on the webpage, so if there's a version of the crazy egg logo that is either square or circular, it may render more correctly.

Reply:
@asaschachar is this logo file better: https://s3.amazonaws.com/resources.crazyegg.com/3-CrazyEgg_Logo.png

Reply:
That other logo still looks rectangular rather than square, but we can try it.